elections/2021-TOC2/README.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Knative TOC Special Election - 2021 Voters Guide"
+linkTitle: "2021 Special Voters Guide"
+weight: 30
+type: "docs"
+---
+
+# 2021 Voters Guide - Knative TOC Special Election
+
+## Purpose
+
+This special election has 2 goals:
+- **Runoff election**
+  - For the 2021 TOC election, we had a tie for one of the 2 TOC seats. This runoff election 
+    is designed to fill the remaining seat due for election this year on the 
+    [Knative Technical Oversight Committee].
+  - This runoff is between 
+    [David Protasowski](candidate-dprotaso.md) and [Scott Nichols](candidate-n3wscott.md).
+  - **Selection Criteria**: Out of these 2 candidates, the one with the most votes in this special election will
```

Comment:
... the most preferred candidate in this special election will get that seat.

"most votes" isn't how preference elections work, so we should be clear on that.